test-integration/student/dashboard.coffee

```diff
@@ -0,0 +1,52 @@
+{describe, CourseSelect, Calendar, ReadingBuilder, StudentDashboard} = require '../helpers'
+{expect} = require 'chai'
+_ = require 'underscore'
+
+
+describe 'Student Dashboard', ->
+
+  @it 'Shows Performance Forecast', ->
+    @dash = new StudentDashboard(@, 'student01', 'Biology I')
```

Comment:
Heh.  That's an oversight.  At one time I had a `beforeEach` with  `@dash ||= new StudentDashboard(...)`

That worked until I wrote the test for creating an assignment as teacher, then navigating back and checking it appeared.  At that point the @dashboard reference wasn't valid any longer since the DOM had been mutated.  So I moved the creation into each spec (and neglected to remove the `@`)

I'll remove the `@`